docs/concepts/packages.md

```diff
@@ -1,113 +1,106 @@
-2. `Token`
-   - Implements the `GRC20` interface.
-   - The underlying struct can be exposed to users. Created with the `Token()` method of `Banker`.
+func Get(key string) int {
+  // tree.Get returns the value at given key in its raw form, 
+  // and a bool to signify the existence of the key-value pair
+  rawValue, exists := tree.Get(key)
+  if !exists {
+	  panic("value at given key does not exist")
+  }
+  
+  // rawValue needs to be converted into the proper type before returning it
+  return rawValue.(int)
+}
+```
 
-## `grc721`
+View the package on the Portal Loop network [here](https://gno.land/p/demo/avl),
+or on GitHub, [here](https://github.com/gnolang/gno/tree/master/examples/gno.land/p/demo/avl).
 
-Gno includes an implementation of the `erc721` non-fungible token standard referred to as `grc721`. The interfaces of `grc721` are as follows:
+### Package `ufmt`
+
+Deployed under `gno.land/p/demo/ufmt`, the `ufmt` package is a minimal version of
+the `fmt` package. From [`ufmt.gno`](https://gno.land/p/demo/ufmt/ufmt.gno):
 
-[embedmd]:# (../assets/explanation/packages/pkg-2.gno go)
 ```go
-// functions that work similarly to those of grc20
-func BalanceOf(owner std.Address) (uint64, error)
-func Approve(approved std.Address, tid TokenID) error
-func TransferFrom(from, to std.Address, tid TokenID) error
-
-// functions unique to grc721
-func OwnerOf(tid TokenID) (std.Address, error)
-func SafeTransferFrom(from, to std.Address, tid TokenID) error
-func SetApprovalForAll(operator std.Address, approved bool) error
-func GetApproved(tid TokenID) (std.Address, error)
-func IsApprovedForAll(owner, operator std.Address) bool
+// Package ufmt provides utility functions for formatting strings, similarly
+// to the Go package "fmt", of which only a subset is currently supported
+// (hence the name µfmt - micro fmt).
+package ufmt
 ```
 
-`grc721` contains a new set of functions:
-
-* `OwnerOf`: Returns the `owner`'s address of a token specified by its `TokenID`.
-* `SafeTransferFrom`: Equivalent to the `TransferFrom` function of `grc20`.
-  * The `Safe` prefix indicates that the function runs a check to ensure that the `to` address is a valid address that can receive tokens.
-  * As you can see from the [code](https://github.com/gnolang/gno/blob/master/examples/gno.land/p/demo/grc/grc721/basic_nft.gno#L341), the concept of `Safe` has yet to be implemented.
-* `SetApprovalForAll`: Approves all tokens owned by the `owner` to an `operator`.
-  * You may not set multiple `operator`s.
-* `GetApproved`: Returns the `address` of the `operator` for a token, specified with its `ID`.
-* `IsApprovedForAll`: Returns if all NFTs of the `owner` have been approved to the `operator`.
+View the package on the Portal Loop network [here](https://gno.land/p/demo/ufmt),
+or on GitHub, [here](https://github.com/gnolang/gno/tree/master/examples/gno.land/p/demo/ufmt).
 
-## `testutils`
+### Package `seqid`
```

Comment:
the "concepts/" doc should not present a specific package but stay high-level and about the concept; the package seqid can be used somewhere else.